fetchmail/fetchmail.py

```diff
@@ -60,7 +60,10 @@ def run(connection, cursor):
             error_message = ""
         except subprocess.CalledProcessError as error:
             error_message = error.output.decode("utf8")
-            print(error.output)
+            # No mail is not an error
+            if not (error_message.startswith("fetchmail: No mail")): 
+                print(fetchmailrc)
```

Comment:
Yes, remember I was trying to debug the access issues.
There is a major flaw here. in case of an error the password is shown in the log.
Its probably wiser to remove the '''print(fetchmailrc)''' statement.